apps/condo/domains/miniapp/schema/B2BAppAccessRight.test.js

```diff
@@ -186,5 +191,64 @@ describe('B2BAppAccessRight', () => {
                 })
             }, NON_SERVICE_USER_ERROR)
         })
+
+        test('One service user can be linked for different B2BApps', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [serviceUser] = await registerNewServiceUserByTestClient(admin)
+
+            const [app] = await createTestB2BApp(admin)
+            const [app2] = await createTestB2BApp(admin)
+
+            await createTestB2BAppAccessRight(admin, serviceUser, app)
+            await createTestB2BAppAccessRight(admin, serviceUser, app2)
+        })
+
+        test('Сan not create or update "B2BAppAccessRight" if "app" and "accessRightSet" refer to different "B2BApp"', async () => {
```

Comment:
It would be looking better inside describe block with caption `Validations` isn't it?

Reply:
Because of here you check code throws an error from `validateInput` hook of the schema

Reply:
The test is inside this block))